src/components/authorise/authorise-get-controller.ts

```diff
@@ -68,6 +79,16 @@ const handleRequestError = (
         `${error.redirectUri}?error=invalid_request&error_description=${encodeURIComponent("Invalid Request")}`
       );
     }
+  } else if (error instanceof ReturnCodeError) {
+    if (config.getReturnCode()) {
+      res.redirect(
+        `${error.redirectUri}?error=invalid_request&error_description=${encodeURIComponent(error.message)}`
+      );
+    } else {
+      res.redirect(
+        `${error.redirectUri}?error=access_denied&error_description=${encodeURIComponent(error.message)}`
```

Comment:
this error description does not match the One Login one - it should be `Access denied by resource owner or authorization server`